web/hooks/organization.go

```diff
@@ -0,0 +1,47 @@
+package hooks
+
+import (
+	"context"
+
+	"github.com/google/go-github/v45/github"
+	"github.com/quickfeed/quickfeed/qf"
+	"github.com/quickfeed/quickfeed/scm"
+)
+
+func (wh GitHubWebHook) handleOrgInvite(e *github.OrganizationEvent) {
+	// check that invitee is enrolled in course
+	invitee := e.GetInvitation().GetLogin()
+	user, err := wh.db.GetUserByCourse(&qf.Course{ScmOrganizationID: uint64(e.GetOrganization().GetID())}, invitee)
```

Comment:
To avoid the somewhat long line; I suggest to do:
```go
org := e.GetOrganization()
course := &qf.Course{
  ScmOrganizationID: uint64(org.GetID()),
  ScmOrganizationName: org.GetLogin(),
}
user, err := wh.db.GetUserByCourse(course, invitee)
```
Then you can reuse `course.GetScmOrganizationName()` below.